instrumentation/log4j/log4j-2.13.2/library/log4j-2.13.2-library.gradle

```diff
@@ -3,5 +3,9 @@ apply from: "$rootDir/gradle/instrumentation-library.gradle"
 dependencies {
   library "org.apache.logging.log4j:log4j-core:2.13.2"
 
+  // Library instrumentation cannot be applied to 2.13.2 due to a bug in Log4J. The agent works
```

Comment:
Didn't expect to find a second case of dependency conflict caused by testing-common, heh